client/lib/plans/features-list.tsx

```diff
@@ -518,34 +519,52 @@ export const FEATURES_LIST: FeatureList = {
 
 	[ FEATURE_PREMIUM_THEMES ]: {
 		getSlug: () => FEATURE_PREMIUM_THEMES,
-		getTitle: () => {
+		getTitle: () => i18n.translate( 'Premium themes' ),
+		getIcon: () => <img src={ Theme2Image } alt={ i18n.translate( 'Premium themes' ) } />,
+		getDescription: () => i18n.translate( 'Switch between a collection of premium design themes.' ),
+		getConditionalTitle: ( planSlug ) => {
+			if ( ! planSlug ) {
+				return '';
+			}
 			const localeSlug = i18n.getLocaleSlug();
-			const shouldShowNewString =
-				( localeSlug && config< string >( 'english_locales' ).includes( localeSlug ) ) ||
-				i18n.hasTranslation( 'Unlimited premium themes' );
-
-			return shouldShowNewString
-				? i18n.translate( 'Unlimited premium themes' )
-				: i18n.translate( 'Premium themes' );
+			const shouldShowInPersonalPlan =
+				config.isEnabled( 'themes/tiers' ) &&
+				( ( localeSlug && config< string >( 'english_locales' ).includes( localeSlug ) ) ||
+					i18n.hasTranslation( 'Dozens of premium themes' ) );
+			if ( shouldShowInPersonalPlan && isPersonalPlan( planSlug ) ) {
+				return i18n.translate( 'Dozens of premium themes' );
+			}
+			if (
+				isPremiumPlan( planSlug ) ||
+				isBusinessPlan( planSlug ) ||
+				isEcommercePlan( planSlug )
+			) {
+				return i18n.translate( 'Unlimited premium themes' );
+			}
+			return '—';
 		},
+	},
+
+	[ WPCOM_FEATURES_PREMIUM_THEMES_UNLIMITED ]: {
+		getSlug: () => WPCOM_FEATURES_PREMIUM_THEMES_UNLIMITED,
+		getTitle: () => i18n.translate( 'Unlimited premium themes' ),
 		getDescription: () => {
 			const localeSlug = i18n.getLocaleSlug();
 			const shouldShowNewString =
 				( localeSlug && config< string >( 'english_locales' ).includes( localeSlug ) ) ||
-				i18n.hasTranslation(
-					'Unlimited access to all of our advanced premium themes, including designs specifically tailored for businesses.'
-				);
-
+				i18n.hasTranslation( 'Switch between all of our premium design themes.' );
 			return shouldShowNewString
-				? i18n.translate(
-						'Unlimited access to all of our advanced premium themes, including designs specifically tailored for businesses.'
-				  )
-				: i18n.translate(
-						'Access to all of our advanced premium theme templates, including templates specifically tailored for businesses.'
-				  );
+				? i18n.translate( 'Switch between all of our premium design themes.' )
+				: i18n.translate( 'Switch between a collection of premium design themes.' );
```

Comment:
Aren't both translations new? Why do we need the `shouldShowTranslation` check in this case?

I'm also not sold on the wording, `premium design themes` sounds strange to me. In the Starter plan section we're just referring to them as `Premium themes`. Would it make sense to use the same naming convention?

Reply:
the "premium design themes" wording has always been there....and it's always sounded weird to me. +1 to not repeating that

Reply:
> Aren't both translations new (I just realized a few lines below this one that it's not the case 😆 )? Why do we need the `shouldShowTranslation` check in this case?

Not sure if you already answered yourself 😅, but to be sure: no, only one of the translations is new:

- `Switch between all of our premium design themes.` -> NEW
- `Switch between a collection of premium design themes.` -> EXISTING

> I'm also not sold on the wording, `premium design themes` sounds strange to me.

I agree, but it's my understanding that @Automattic/martech controls the language used in the plans grid, and any change in the copy should be handled by them since the impact of such change should be carefully measured.

So that's why I chose to stick with the existing "premium design themes" terminology.